kmod/scoutfs-kmod.spec.in

```diff
@@ -111,8 +111,8 @@ SCOUTFS_RPM_NAME=$(rpm -q %{name} | grep "%{version}-%{release}")
 rpm -ql $SCOUTFS_RPM_NAME | grep '\.ko$' > /var/run/%{name}-modules-%{version}-%{release} || true
 
 %postun
-if [ -x /sbin/weak-modules ]; then
-    cat /var/run/%{name}-modules-%{version}-%{release} | /sbin/weak-modules --remove-modules --no-initramfs
+if [ -x "$(which weak-modules)" ]; then
```

Comment:
"which" already tests for being executable, and $() doesn't eat stderr so this still leaks error messages from which when weak-modules isn't found.  If we wanted to use which and silence output, just do that.   But do we have to have this weak-modules invocation conditional at all?  Why here and not earlier in the same spec?  If no one can answer that, just drop the conditional.  If we're not addressing the unconditional invocation above then changing it here makes it symmetrical, at least.  Someone may still have to fix it in the future either way.

Reply:
I can't answer that and thus will drop it 😎 Might be a relic from a bygone era when weak-modules wasn't a default. Who knows, easy fix.

Reply:
Agreed, who knows. I assume it's copy-pasted fragments originating from something trying to be an all-in-one spec file that can do dkms, weak-modules, etc, depending on what's present.  The pattern's found in a whole bunch of external spec files.